rosetta-cli-conf/testnet/config.json

```diff
@@ -40,7 +40,6 @@
     "bootstrap_balances": "",
     "interesting_accounts": "",
     "reconciliation_disabled": false,
-    "inactive_discrepency_search_disabled": false,
```

Comment:
Why remove this line?

Reply:
It's no longer supported in rosetta-cli

Reply:
wuhhhhh. that was hard to make

Reply:
This was a correction in spelling `inactive_discrepency_search_disabled` -> `inactive_discrepancy_search_disabled`. I am wondering how to make it more clear while upgrading the cli packages since I am planning more config changes to make that config more clear and consistent.